$OpenBSD$
# Drop the tls-session-manager fields removed by the older tls and auto-update APIs.
--- tls-session-manager-0.0.8/Network/TLS/SessionManager.hs
+++ tls-session-manager-0.0.8/Network/TLS/SessionManager.hs
@@ -98,7 +98,6 @@
                 , reaperAction = clean
                 , reaperNull = Q.null
                 , reaperDelay = pruningDelay conf * 1000000
-                , reaperThreadName = "TLS session manager"
                 }
     return $
         noSessionManager
@@ -106,7 +105,6 @@
             , sessionResumeOnlyOnce = resume reaper SingleUse
             , sessionEstablish = \x y -> establish reaper lifetime x y >> return Nothing
             , sessionInvalidate = invalidate reaper
-            , sessionUseTicket = False
             }
